Week1/MAKEME.md

```diff
@@ -7,18 +7,23 @@
 
 ### 1. Testing Exercises
 
+Using **Slack** application on desktop and mobile:
+
+- Describe shared testing activities for desktop and mobile versions
+
 ### 2. Assignment
 
-Given an application with web, desktop and mobile versions (e.g. WhatsApp):
+Given [RealWorld example application](https://github.com/gothinkster/realworld) on web ([React/Redux](https://github.com/gothinkster/react-redux-realworld-example-app)) and on mobile ([NativeScript (Angular)](https://github.com/nea/nativescript-realworld-example-app))
 
-- Describe shared testing activities for desktop and mobile
-- Describe unique testing activities per platform
-- Give two possible suggestions to improve the quality of the application
+- Describe shared testing activities for web and mobile
+- Describe unique testing activities per platform (web and mobile)
+- Come up with two possible suggestions to improve the quality of the application
 
 ### Finished?
 
 At the end of this week you need to deliver upon the following:
 
-- ...
+- Create a Markdown document under Week1 inclduing the description for shared and unique testing activities for [RealWorld](https://react-redux.realworld.io/#/?_k=bkdto8) web and mobile apps (you may have create a list or a table). Then create a PR to be reviewed
```

Comment:
Maybe you can be more specific in how students would submit the homework, i.e. fork this repo and make a PR with their homework? You can, of course, also mention that during the Q&A today.

Reply:
there is "Create PR to be reviewed at the end" but I forgot to add the "fork" bit :)